packages/nodes-base/nodes/Wait/Wait.node.ts

```diff
@@ -294,6 +302,29 @@ export class Wait extends Webhook {
 				default: 'timeInterval',
 				description: 'Determines the waiting mode to use before the workflow continues',
 			},
+			{
```

Comment:
there seems to be a bit of duplication between this and the next property. is this because the auth options are more limited here?

Reply:
yes, we probably should avoid sharing specialized node's properties between different node types as this creates some coupling